mapping/jsonld-contexts/mapping.jsonld

```diff
@@ -19,6 +19,7 @@
 		"qoS": "https://ontology.eglobalmark.com/qoS",
 		"subAttribute1": "https://ontology.eglobalmark.com/subAttribute1",
 		"subAttribute2": "https://ontology.eglobalmark.com/subAttribute2",
-		"subAttribute3": "https://ontology.eglobalmark.com/subAttribute3"
+		"subAttribute3": "https://ontology.eglobalmark.com/subAttribute3",
+		"specificAccessPolicy": "https://ontology.eglobalmark.com/egm#specificAccessPolicy"
```

Comment:
too bad to duplicate a term that is already defined in the EGM cross-context :(

Reply:
This was why I did it: [https://github.com/easy-global-market/ngsild-api-data-models/pull/66#discussion_r701688413](https://github.com/easy-global-market/ngsild-api-data-models/pull/66#discussion_r701688413)

Reply:
Adding `specificAccessPolicy ` to each context is not the best way, but in the other hand adding all the egm context may causes several issues since it defines  many other concepts.